src/components/views/elements/ImageView.tsx

```diff
@@ -190,14 +199,14 @@ export default class ImageView extends React.Component<IProps, IState> {
 
     private onRotateCounterClockwiseClick = () => {
         const cur = this.state.rotation;
-        const rotationDegrees = cur - 90;
-        this.setState({ rotation: rotationDegrees });
+        this.rotation = cur - 90;
+        this.calculateZoomAndRotate();
```

Comment:
this feels like it ought to just be an optional parameter to the method rather than now having a confusing property and state field